swagger.yaml

```diff
@@ -195,6 +195,8 @@ definitions:
         type: integer
       request_num_samples:
         type: integer
+      ca:
```

Comment:
caCert will be a bit more clear name for this attribute, and if also CaSecretName --> CaCertSecretName

Reply:
come on.....CA is always a certificate. It is by definition a certificate. Why burden with so much text?

Reply:
@ydayagi I agree with @masayag 
I think it's a terminology issue - you mentioned CA as a certificate but the terminology that we used in the code (as I know) is ca certificate